scss/components/input-group.scss

```diff
@@ -135,7 +135,13 @@ $block: #{$fd-namespace}-input-group;
         &--button {
           padding: 0;
           [class*="fd-button"] {
-            border-radius: 0;
+            border-top-right-radius: 3px;
+            border-bottom-right-radius: 3px;
```

Comment:
Is there a variable for this radius?  Seems like it would be used in a bunch of places.

Reply:
@greg-a-smith Yes, you are right. Changes will be made.